### ISSUE TITLE
Ability to select objects that are under the top one

### ISSUE THREAD

**Issue by rezanid** [jgraph/drawio-desktop#482]
* [X] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [X] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
When a diagram has several items that are put on top of each other (very common in software diagrams e.g. UML and ArchiMate) it is frustrating to to try to select an object that is not the top one. One has to reshuffle the objects, or move them temporarily just to be able to select and change another one that is under. 

**Describe the solution you'd like**
One technique that I have seen some vector drawing software use is switching between objects if the user continues clicking without changing the mouse position.
Imagine there are three object on top of each other. Object 1 is on top of 2 and object 2 is on top of 3. When user click on object 1 on a position that is also relevant to object 2 and 3, by default the object 1 is selected. If user clicks another time without moving the mouse pointer, then object 2 is selected and then object 3.

**Describe alternatives you've considered**
Another alternative (though I haven't seen it in another app) is to let user select among different objects, if the user clicks and hold for a few seconds. In other words, in above scenario, when user clicks on object 1 and continues holding the button for a few seconds a context menu could display a list of all the relevant objects for selection. When user hovers the mouse over an object name, it can be highlighted. If user then clicks on the object name, the object will be selected and context menu would disappear.

**Additional context**
I think that the first solution would be easier to implement and having personally worked with it (I wish I remember which app, I guess Microsoft Expression Design) I think it's pretty natural and easy from user perspective.

**Comment by mararad** [comment]
Hi,

thanks for the suggestion. You could try to select all descendants (right mouse click or Alt+Shift+d) as some kind of a workaround.

**Comment by rezanid** [comment]
Hi @mararad,

Thanks for replying. The shapes that I talk about are not grouped, and I don't find an option to allow me to select descendants when I right-click. Pressing Alt+Shift+D while the top shape is selected does not do anything either.

**Comment by rezanid** [comment]
The following is an example of multiple shapes stacked over each other.
![image](https://user-images.githubusercontent.com/14060208/106424024-cd114b00-6461-11eb-825a-f259b90ec70b.png)

**Comment by mararad** [comment]
Yeah, select descendants won't work in your case. You might want to try tab key but not sure if it suits your needs.

**Comment by timblaktu** [comment]
@davidjgraph Why was this closed as Completed? I see no resolution, not even a "won't fix". Can you provide some clarity?

@rezanid have you found any new acceptable workarounds? The only thing I have found is to move the top container out of the way so it's no longer "masking" the objects it contains. 

This is the default behavior of many container objects from the AWS shape libraries, e.g. Account, Region, VPC, etc.

There is no grouping in my scenario, and I have tried tweaking these objects' properties (container, moveable, etc.) to no avail.

**Comment by davidjgraph** [comment]
It has zero votes.

**Comment by rezanid** [comment]
@timblaktu unfortunately not. I think most people don't think there can be a solution for this annoying problem they just struggle with it and think that's the way it is.

**Comment by rikka0w0** [comment]
We really need this feature

**Comment by davidjgraph** [comment]
Still has no votes...

**Comment by 519seven** [comment]
I'd like to vote for this.  The workaround is to make the object below hang out to the side so it's selectable but you have to make sure to change the shape while you can still access it.  It can get tricky.

**Comment by nitedani** [comment]
I don't use drawio because of this issue.
